### ISSUE TITLE
How to: Fix usage with Preact

### ISSUE THREAD

**Issue by philmetzger** [DominicTobias/react-image-crop#509]
I ran into this issue on production:
`TypeError: Cannot read properties of undefined (reading 'current')`

And noticed number 16 here noting the issue:
https://bestofreactjs.com/repo/DominicTobias-react-image-crop-react-image-manipulation

However I am not sure how to actually fix this?

I am using NextJS and my `next.config.js` looks like this:
```
webpack: (config, { dev, isServer }) => {
    config.module.rules.push({
      test: /\.md$/,
      use: 'raw-loader',
    });

    if (!dev && !isServer) {
      Object.assign(config.resolve.alias, {
        react: 'preact/compat',
        'react-dom': 'preact/compat',
      });
    }

    return config;
  },
```

and tsconfig.json:
```
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "baseUrl": "./src",
    "paths": {
      "@/*": ["./*"],
    },
    "incremental": true,
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx"],
  "exclude": ["node_modules"]
}

```

Any tips?

**Comment by g5l** [comment]
I'm having the same issue with preact.

**Comment by DominicTobias** [comment]
Hi, can you post the full error trace/what line of code the exception
happens?

**Comment by philmetzger** [comment]
Sure hope this is what you are looking for:

i```
nstrument.js:107 TypeError: Cannot read properties of undefined (reading 'current')
    at l (ReactCrop.min.js:2:821)
    at m.render (ReactCrop.min.js:2:13884)
    at j (preact.module.js:1:5807)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
    at j (preact.module.js:1:6320)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
    at j (preact.module.js:1:6320)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
    at j (preact.module.js:1:6320)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
    at j (preact.module.js:1:6320)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
    at j (preact.module.js:1:6320)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
    at j (preact.module.js:1:6320)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at j (preact.module.js:1:6138)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
    at j (preact.module.js:1:6320)
    at S (preact.module.js:1:2137)
    at L (preact.module.js:1:7356)
```

**Comment by DominicTobias** [comment]
This should be fixed in 10.0.8
